src/DmaRequestCore.bsv

```diff
@@ -6,11 +6,11 @@ import DmaTypes::*;
 
 
 typedef 4096                                BUS_BOUNDARY;
-typedef TLog#(BUS_BOUNDARY)                 BUS_BOUNDARY_WIDTH;
+typedef TLog#(TAdd#(1, BUS_BOUNDARY))       BUS_BOUNDARY_WIDTH;
```

Comment:
We would rather use `TAdd#(1, TLog#(BUS_BOUNDARY))` instead.  The meaning of this is a lot more clear

Reply:
Get it.